meta-mentor-staging/recipes-core/udev/udev-extraconf_%.bbappend

```diff
@@ -2,6 +2,7 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI +=  "file://0001-udev-extraconf-mount.sh-add-LABELs-to-mountpoints.patch \
              ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'file://systemd-udevd.service', '', d)} \
              file://0001-udev-extraconf-mount.sh-define-mount-prefix-using-a-.patch \
+             file://0001-udev-extraconf-mount.sh-ignore-lvm-in-automount.patch \
```

Comment:
please add the patch at the end i.e. after the next two patches so the patch series are consolidated.